finalPrep/modZeroHero.js

```diff
@@ -55,30 +79,74 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
-
+var scaryMonster = {
+  name = "Teke Teke",
+  smell = "Death",
+  weight = 3,
+  citiesDestroyed = ["London", "Paris", "New York"],
+  luckyNumbers = [3, 7, 13],
+  address: {
+    number: 13,
+    street: "Evil Street",
+    state: "FL",
+    zip:30315};
```

Comment:
Usually I would keep this closing curly bracket on the next line without a semicolon after it.